docs/deployment/pull-backup.rst

```diff
@@ -301,3 +301,108 @@ a backup may be the following command::
 
 This command also automatically removes the socket file after the ``borg
 create`` command is done.
+
+ssh-agent
+=========
+
+In this scenario *borg-server* initiates an SSH connection to *borg-client* and forwards the authentication
+agent connection.
+
+After that, it works similar to the push mode:
+*borg-client* initiates another SSH connection back to *borg-server* using the forwarded authentication agent
+connection to authenticate itself, starts ``borg serve`` and communicates with it.
+
+Using this method requires ssh access of user *borgs* to *borgc@borg-client*, where:
+
+* *borgs* is the user on the server side with read/write access to local borg repository.
+* *borgc* is the user on the client side with read access to files meant to be backed up.
+
+Applying this method for automated backup operations
+----------------------------------------------------
+
+Assume that the borg-client host is untrusted.
+Therefore we do some effort to prevent a hostile user on the borg-client side to do something harmful.
+In case of a fully trusted borg-client the method could be simplified.
+
+Preparing the server side
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Do this once for each client on *borg-server* to allow *borgs* to connect itself on *borg-server* using a
+dedicated ssh key:
+
+::
+
+  borgs@borg-server$ install -m 700 -d ~/.ssh/
+  borgs@borg-server$ ssh-keygen -N '' -t rsa  -f ~/.ssh/borg-client_key
+  borgs@borg-server$ { echo -n 'command="borg serve --append-only --restrict-to-repo ~/repo",restrict '; cat ~/.ssh/borg-client_key.pub; } >> ~/.ssh/authorized_keys
+  borgs@borg-server$ chmod 600 ~/.ssh/authorized_keys
+
+``install -m 700 -d ~/.ssh/``
+
+  Create directory ~/.ssh with correct permissions if it does not exist yet.
+
+``ssh-keygen -N '' -t rsa  -f ~/.ssh/borg-client_key``
+
+  Create an ssh key dedicated to communication with borg-client.
+
+.. note::
+  Another more complex approach is using a unique ssh key for each pull operation.
+  This is more secure as it guarantees that the key will not be used for other purposes.
+
+``{ echo -n 'command="borg serve --append-only --restrict-to-repo ~/repo",restrict '; cat ~/.ssh/borg-client_key.pub; } >> ~/.ssh/authorized_keys``
+
+  Add borg-client's ssh public key to ~/.ssh/authorized_keys with forced command and restricted mode.
+  The borg client is restricted to use a repo path below the specified path and to append-only operation.
```

Comment:
as you've changed the command to --restrict-to-repo, the description needs changing also:

The borg client is restricted to use one repo at the specified path and to append-only operation.

Reply:
Fixed.